<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenDevoir</class>
 <widget class="QDialog" name="FenDevoir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Devoirs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Devoirs disponibles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="devoirs"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Liste des matières disponibles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="matieresDisponibles"/>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="SendHomeworkRequest">
         <property name="text">
          <string>Demander les devoirs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="requestMatieres">
         <property name="text">
          <string>Demander les matières disponibles pour ma classe</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Chat</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTextEdit" name="zoneMessage"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="message"/>
         </item>
         <item>
          <widget class="QPushButton" name="SendMessage">
           <property name="text">
            <string>Envoyer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Contrôle administrateur(nécessite des droits spéciaux)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="CreateAccount">
         <property name="text">
          <string>Créer un compte</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="DeleteAccount">
         <property name="text">
          <string>Supprimer un compte</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="CreateDevoir">
         <property name="text">
          <string>Créer un devoir</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="DeleteDevoir">
         <property name="text">
          <string>Supprimer un devoir</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTableView" name="clientsConnectes"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="sendSystemMessage">
         <property name="text">
          <string>Envoyer un message système</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
